systems/plants/TimeSteppingRigidBodyManipulator.m

```diff
@@ -126,7 +126,7 @@ function checkDirty(obj)
 
     function model = compile(model)
       w = warning('off','Drake:RigidBodyManipulator:UnsupportedContactPoints');
-      model.gurobi_present = checkDependency('gurobi');
+      model.gurobi_present = true;
```

Comment:
This is due to merging in mitdrc/master at some point. @rdeits is handling this.